src/Simplify/AstHelpers.elm

```diff
@@ -23,3 +25,16 @@ removeParensFromPattern node =
 
         _ ->
             node
+
+
+moduleHeaderExposing : Elm.Syntax.Module.Module -> Node Exposing
```

Comment:
What's the idea behind the `AstHelper` module like only containing AST transformations? If not I'd probably want move most of the matchers and parsers from `Simplify` there.

Reply:
To answer your question, I guess we could split the helpers a bit more yes. That would probably help make them more findable in practice?

But for this function, there is the `Elm.Module.Exposing.exposingList` which already does the same thing (well, it doesn't return a `Node`, but you seem to not care about that anyway)